### PR TITLE
Fixed bug during node creation in edgesite

### PR DESCRIPTION
**Pull request by antoniomuso**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Fixed bug during node creation in edgesite.
```console
register node failed, error: bad request module name(websocket)
```

**Which issue(s) this PR fixes**:

Fixes [#2026](https://github.com/kubeedge/kubeedge/issues/2026)

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Review by fisherxu — COMMENTED** [review]
Thanks for the fixing, @antoniomuso

File: edge/pkg/edged/edged_status.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ package edged
 
 import (
 	"fmt"
+	"github.com/kubeedge/beehive/pkg/core"
```
Comment (fisherxu) [inline edge/pkg/edged/edged_status.go:27]: Please sorts the import here.

Comment (antoniomuso) [reply edge/pkg/edged/edged_status.go]: Ok I sort the import.

Comment (fisherxu) [reply edge/pkg/edged/edged_status.go]: Thanks, would you like to squash the commits?

Comment (antoniomuso) [reply edge/pkg/edged/edged_status.go]: Done

**Pull request merged**